src/globus_sdk/experimental/login_flow_manager/command_line_login_flow_manager.py

```diff
@@ -10,6 +10,15 @@ class CommandLineLoginFlowManager(LoginFlowManager):
     """
     A ``CommandLineLoginFlowManager`` is a ``LoginFlowManager`` that uses the command
     line for interacting with the user during its interactive login flows.
+
+    Example usage:
+
+    >>> login_client = globus_sdk.NativeAppAuthClient(...)
+    >>> login_flow_manager = CommandLineLoginFlowManager(login_client)
+    >>> scopes = [globus_sdk.scopes.TransferScopes.all]
+    >>> auth_params = GlobusAuthorizationParameters(required_scopes=scopes)
+    >>> tokens = login_flow_manager.run_login_flow(auth_params)
```

Comment:
Is this extracted by a Sphinx extension? If so, it needs this formatting:

```suggestion
    ..  code-block:: pycon

        >>> login_client = globus_sdk.NativeAppAuthClient(...)
        >>> login_flow_manager = CommandLineLoginFlowManager(login_client)
        >>> scopes = [globus_sdk.scopes.TransferScopes.all]
        >>> auth_params = GlobusAuthorizationParameters(required_scopes=scopes)
        >>> tokens = login_flow_manager.run_login_flow(auth_params)
```

Reply:
I don't think it is? I'm not seeing that formatting anywhere else in the SDK at least

Reply:
Built it locally, and you're correct. Thanks!